compiler/luci-interpreter/src/kernels/Equal.h

```diff
@@ -42,9 +42,9 @@ class Equal : public Kernel
 
 private:
   int32_t _x_multiplier = 0;
-  int32_t _x_shift = 0;
+  int _x_shift = 0;
   int32_t _y_multiplier = 0;
-  int32_t _y_shift = 0;
+  int _y_shift = 0;
```

Comment:
Why we need this changes? Datatype should be less specific?

Reply:
This is because of it's usage in quantizeMultiplierSmallerThanOneExp function:
it takes `int&` and it is incompatible with `int32_t &`

Another option is to rewrite this function, but I looked at tensorflow implementation and found that they use simple int, so I choose it =)